### ISSUE TITLE
Support for SteamVR?

### ISSUE THREAD

**Issue by ShacharWeis** [Unity-Technologies/XR-Interaction-Toolkit-Examples#6]
SteamVR is not on the list of XR Plugins, is it coming soon?

**Comment by sivatejachigicherla** [comment]
https://blogs.unity3d.com/2020/01/24/unity-xr-platform-updates/
Unity may make OpenVR as a replacement for SteamVR

**Comment by ShacharWeis** [comment]
No, they are waiting for Valve to make their own OpenVR plugin. 

Unity made an in-house plugin for Magic Leap, which has less than 5000 headsets on the market, but not for OpenVR, which has 500,000 headsets. I'm guess the reason is money and politics.

**Comment by enqui** [comment]
It may also be that OpenVR forces users to install Steam in order to use VR and Unity wants to provide an XR platform that is truly Open and cross platform, not just by name.  It's probable that friction is there too as Valve needs to defend itself and create alternate growth channels against the competition from Epic. I suppose we will never find out. 

What is important as new projects usually start in March, is to get a useful date (i.e. not in six months,) for the new plugin from Valve because if there is none, there are two options: 

A. Not use Unity from now on
B. Not use OpenVR from now on

And let's be honest, divesting an entire studio from its primary technology platform investment sounds more unlikely and high risk than switching VR platform...

**Comment by HeadClot** [comment]
I really wish Unity just made an OpenVR plugin opposed to a Magic Leap plugin.

**Comment by Corysia** [comment]
https://steamcommunity.com/app/250820/discussions/7/2268069450205612646/
